src/app/Components/Header/Components/HeaderOption/index.snap.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import HeaderOption from '.';
+import search from '../../assets/search.png';
+
+test('It matches the snapshot', () => {
```

Comment:
Even though this is syntactically correct, we tend to use:
`it ('matches the snapshot', () => ....`
Since it's more readable and natural. This applies to all the tests you currently have.